vividus/src/main/java/org/vividus/configuration/AbstractPropertiesProcessor.java

```diff
@@ -23,13 +23,16 @@
 
 public abstract class AbstractPropertiesProcessor implements PropertiesProcessor
 {
-    private final String processorMarker;
+    private static final String VALUE_GROUP = "\\((.+?)\\)";
+    private static final String BRACKETS = "(%s)";
+
+    private final String processorRegexMarker;
     private final Pattern propertyPattern;
 
-    AbstractPropertiesProcessor(String processorMarker)
+    AbstractPropertiesProcessor(String processorRegexMarker)
     {
-        this.processorMarker = processorMarker;
-        this.propertyPattern = Pattern.compile(processorMarker + "\\((.+?)\\)");
+        this.processorRegexMarker = processorRegexMarker;
+        this.propertyPattern = Pattern.compile(String.format(BRACKETS, processorRegexMarker) + VALUE_GROUP);
```

Comment:
why is it needed to wrap `processorRegexMarker` into group?

Reply:
for correct work of regex for VAULT, didn't find better solution